TELCODOCS-2045 - Telco core RDS 4.18 docs (#88622)
openshift/openshift-docs: modules/telco-core-topology-aware-lifecycle-manager.adoc

// Module included in the following assemblies:
//
// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc

:_mod-docs-content-type: REFERENCE
[id="telco-core-topology-aware-lifecycle-manager_{context}"]
= Topology Aware Lifecycle Manager

New in this release::
No reference design updates in this release.

Description::

> {cgu-operator-full} is an Operator that runs only on the hub cluster for managing how changes including cluster and Operator upgrades, configuration, and so on are rolled out to the network.

Reviewer: Not sure what this means. 

```suggestion
{cgu-operator-full} is an Operator, which runs only on the hub cluster, manages how changes, including cluster and Operator upgrades, configuration, and so on, are rolled out to the network.
```

> 

Reviewer: List needs a lead-in sentence.
